kafka/monitoring-kafka-performance-metrics.md

```diff
@@ -9,7 +9,8 @@ There are a few key differences between Kafka and other queueing systems like [R
 3. It is [very fast][kafka-benchmark], even in a small cluster. 
 4. It has strong ordering semantics and durability guarantees.
 
-Despite being pre-1.0, (current version is [0.9.0.1][download-kafka]), it is [production-ready][user-list], and powers a large number of high-profile companies including [LinkedIn][linkedin-msgs], Yahoo, Netflix, and Datadog.
+Kafka is [production-ready][user-list], and [powers a large number of high-profile companies][user-list] including [LinkedIn][linkedin-msgs], Yahoo, Netflix, and Datadog.  
+The information presented in this article has been updated with information for Kafka 0.9+, however metrics may have changed in newer releases. Refer to the Kafka [downloads page][download-kafka] for the latest version, and the key metrics section below for a link to the Kafka metrics documentation.    
```

Comment:
I'd rather not add this disclaimer if it's not clear that any metrics have changed.

Reply:
I could go either way, though for someone who isn't able to find the mentioned metrics on this page in the newer versions, this diclaimer seems useful.

Reply:
I can do a quick `jmxterm`, or whatever, if you think a cross-check makes sense.

Reply:
@cricket007 If there's a way to do a quick cross-check, that would be great. But if not, we can do a more comprehensive sweep through the post (just might take a little time)